sdk/appcenter-push/src/main/java/com/microsoft/appcenter/push/Push.java

```diff
@@ -313,12 +320,24 @@ public synchronized void onStarted(@NonNull Context context, @NonNull Channel ch
         mAuthListener = new AbstractTokenContextListener() {
 
             @Override
+            // TODO check synchronized
+            // TODO post to background thread?
```

Comment:
The listener is already called from background thread in practice.
mChannel is non null after start so no need for synchronized.